l10n/src/main.ts

```diff
@@ -143,6 +143,7 @@ export function t(strs: TemplateStringsArray, ...replacements: L10nReplacement[]
  * value will be returned (with injected args values for any templated values).
  * @param options - The options to use when localizing the message.
  * @returns localized string with injected arguments.
+ * @example `l10n.t({ message: 'Hello {name}', args: { name: 'Erich' }, comment: 'This is a comment' } );`
```

Comment:
```suggestion
 * @example `l10n.t({ message: 'Hello {0}', args: ['Erich'], comment: 'This is a comment' } );`
```

I'd prefer this syntax to be used in the example.